src/main.rs

```diff
@@ -57,6 +61,7 @@ fn main() -> Result<()> {
         settings.root, settings.extensions, settings.ignore_dirs
     );
     let mut walked = 0;
+    
     let files = WalkDir::new(&settings.root)
         .into_iter()
         .filter_entry(|e| !e.file_type().is_dir() || !settings.ignore_dirs.contains(e.file_name()))
```

Comment:
need to change here to support regex, as some ignore files uses regex expressions